### PR TITLE
ADD: Edit profile

### PR DESCRIPTION
**Pull request by co012**
Screenshot 2022-09-07 at 09 11 48
Screenshot 2022-09-07 at 09 12 01
Screenshot 2022-09-07 at 09 11 34
Screenshot 2022-09-07 at 09 11 28

The ability to change profile picture will be added in a separated pull
Display errors, success too in another pull

### CODE REVIEW

**Comment by co012** [comment]
@alejomendoza, how should display errors, success, and loading state?

File: packages/app/features/edit-profile/uploadProfilePictureBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { Text, View } from "app/design-system";
+import React from "react";
+
+export function UploadProfilePictureBanner() {
```
Comment (jmysliv) [inline packages/app/features/edit-profile/uploadProfilePictureBanner.tsx:4]: do we need a separate component for it?

Comment (co012) [reply packages/app/features/edit-profile/uploadProfilePictureBanner.tsx:4]: Probably not, but I would argue that this code is easier to read.

Comment (jmysliv) [reply packages/app/features/edit-profile/uploadProfilePictureBanner.tsx:4]: In my opinion, it's so small that there's no point in separating it.

**Review by jmysliv — APPROVED** [review]
Fix that one thing and you can merge 🙂

**Review by jmysliv — APPROVED** [review]

**Pull request merged**